k8s.gcr.io/images/k8s-staging-sig-storage/images.yaml

```diff
@@ -99,7 +100,7 @@
     "sha256:6029c252dae6178c99b580de72d7776158edbc81be0de15cedc4152a3acfed18": [ "v1.7.3" ]
 - name: iscsiplugin
   dmap:
-    "sha256:31117074c183242b223340f2fc0f886243e97c6cb1a83d98cfb00a69d1746697": [ "v0.1.0" ]
+    "sha256:8ff2016fd563f0365ccf6ad4ab09f09a1941cbede5c62c5692e95589c0ed41de": [ "v0.1.0" ]
```

Comment:
this is a little bit concerning, the image must have been created again

Reply:
error log from `pull-k8sio-cip`

```
level=fatal msg="run `cip run`: promoting images: [edge &{{gcr.io/k8s-staging-sig-storage   true} {iscsiplugin v0.1.0} sha256:8ff2016fd563f0365ccf6ad4ab09f09a1941cbede5c62c5692e95589c0ed41de {asia.gcr.io/k8s-artifacts-prod/sig-storage k8s-infra-gcr-promoter@k8s-artifacts-prod.iam.gserviceaccount.com  false} {iscsiplugin v0.1.0}}: tag 'v0.1.0' in dest points to sha256:31117074c183242b223340f2fc0f886243e97c6cb1a83d98cfb00a69d1746697, not sha256:8ff2016fd563f0365ccf6ad4ab09f09a1941cbede5c62c5692e95589c0ed41de (as per the manifest), but tag moves are not supported; skipping edge &{{gcr.io/k8s-staging-sig-storage   true} {iscsiplugin v0.1.0} sha256:8ff2016fd563f0365ccf6ad4ab09f09a1941cbede5c62c5692e95589c0ed41de {eu.gcr.io/k8s-artifacts-prod/sig-storage k8s-infra-gcr-promoter@k8s-artifacts-prod.iam.gserviceaccount.com  false} {iscsiplugin v0.1.0}}: tag 'v0.1.0' in dest points to sha256:31117074c183242b223340f2fc0f886243e97c6cb1a83d98cfb00a69d1746697, not sha256:8ff2016fd563f0365ccf6ad4ab09f09a1941cbede5c62c5692e95589c0ed41de (as per the manifest), but tag moves are not supported; skipping edge &{{gcr.io/k8s-staging-sig-storage   true} {iscsiplugin v0.1.0} sha256:8ff2016fd563f0365ccf6ad4ab09f09a1941cbede5c62c5692e95589c0ed41de {us.gcr.io/k8s-artifacts-prod/sig-storage k8s-infra-gcr-promoter@k8s-artifacts-prod.iam.gserviceaccount.com  false} {iscsiplugin v0.1.0}}: tag 'v0.1.0' in dest points to sha256:31117074c183242b223340f2fc0f886243e97c6cb1a83d98cfb00a69d1746697, not sha256:8ff2016fd563f0365ccf6ad4ab09f09a1941cbede5c62c5692e95589c0ed41de (as per the manifest), but tag moves are not supported; skipping]"
```

Reply:
Opened https://github.com/kubernetes-csi/csi-driver-iscsi/issues/97 to track this

Reply:
done